javascript/ql/src/semmle/javascript/Expr.qll

```diff
@@ -1288,6 +1288,15 @@ class LogOrExpr extends @logorexpr, BinaryExpr {
   override ControlFlowNode getFirstControlFlowNode() { result = this }
 }
 
+/** A nullish coalescing '??' expression. */
+class NullishCoalescingExpr extends @nullishcoalescingexpr, BinaryExpr {
```

Comment:
Perhaps also add this to `LogicalBinaryExpr` while you're at it.

Reply:
I think we decided to do this after all (cf. Asger's latest comment), which will allow us to get rid of the changes in `DataFlow.qll` (above and below) and `TypeInference.qll` (below).

Reply:
Done.